docker-compose.yml

```diff
@@ -74,97 +69,10 @@ services:
       - "8887:8887"
       - "1700:1700/udp"
 
-    environment:
-      # If using CockroachDB:
-      - TTN_LW_IS_DATABASE_URI=postgres://root@cockroach:26257/${DEV_DATABASE_NAME:-ttn_lorawan}?sslmode=disable
-      # If using PostgreSQL:
-      # - TTN_LW_IS_DATABASE_URI=postgres://root@postgres:5432/${DEV_DATABASE_NAME:-ttn_lorawan}?sslmode=disable
-      - TTN_LW_REDIS_ADDRESS=redis:6379
-
-      # Let's Encrypt for "thethings.example.com":
-      # - TTN_LW_TLS_SOURCE=acme
-      # - TTN_LW_TLS_ACME_DIR=/var/lib/acme
-      # - TTN_LW_TLS_ACME_EMAIL=you@thethings.example.com
-      # - TTN_LW_TLS_ACME_HOSTS=thethings.example.com
-      # - TTN_LW_TLS_ACME_DEFAULT_HOST=thethings.example.com
-
-      # If using (self) signed certificates:
-      - TTN_LW_TLS_SOURCE=file
-      - TTN_LW_TLS_ROOT_CA=/run/secrets/cert.pem
-      - TTN_LW_TLS_CERTIFICATE=/run/secrets/cert.pem
-      - TTN_LW_TLS_KEY=/run/secrets/key.pem
-
-      # HTTP Server configuration:
-      - TTN_LW_HTTP_COOKIE_HASH_KEY # generate 64 bytes (openssl rand -hex 64)
-      - TTN_LW_HTTP_COOKIE_BLOCK_KEY # generate 32 bytes (openssl rand -hex 32)
-      - TTN_LW_HTTP_METRICS_PASSWORD # choose a password
-      - TTN_LW_HTTP_PPROF_PASSWORD # choose a password
-
-      # Email configuration for "thethings.example.com":
-      # - TTN_LW_IS_EMAIL_SENDER_NAME=The Things Stack
-      # - TTN_LW_IS_EMAIL_SENDER_ADDRESS=noreply@thethings.example.com
-      # - TTN_LW_IS_EMAIL_NETWORK_CONSOLE_URL=https://thethings.example.com/console
-      # - TTN_LW_IS_EMAIL_NETWORK_IDENTITY_SERVER_URL=https://thethings.example.com/oauth
-
-      # If sending email with Sendgrid:
-      # - TTN_LW_IS_EMAIL_PROVIDER=sendgrid
-      # - TTN_LW_IS_EMAIL_SENDGRID_API_KEY=enter Sendgrid API key
-
-      # If sending email with SMTP:
-      # - TTN_LW_IS_EMAIL_PROVIDER=smtp
-      # - TTN_LW_IS_EMAIL_SMTP_ADDRESS=enter mail server address
-      # - TTN_LW_IS_EMAIL_SMTP_USERNAME=enter username
-      # - TTN_LW_IS_EMAIL_SMTP_PASSWORD=enter password
-
-      # If Application Server enabled, defaults for "thethings.example.com":
-      # - TTN_LW_AS_MQTT_PUBLIC_ADDRESS=thethings.example.com:1883
-      # - TTN_LW_AS_MQTT_PUBLIC_TLS_ADDRESS=thethings.example.com:8883
-
-      # If Application Server enabled, Application Server interoperability configuration:
```

Comment:
Why is this removed and where is this documentation moved?

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/1980#issuecomment-587091137